src/commands/createFunction/javaSteps/JavaFunctionCreateStep.ts

```diff
@@ -2,15 +2,14 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
> Yes, I agree that GitHub link is not a stable solution for function template, I've revert the changes for Java template provider but just update the FunctionCreationStep, will update the template provider once we add java templates to template repository.

Just to clarify - do you want this merged with just the `FunctionCreationStep` stuff or do you want to wait until you can update the template provider? We're about to do a release of the extension so anything merged in the next day or two will be included in that release

Reply:
I think we could release this PR first as a perf enhancement for Java, and I'll update the template provider later.